WildAidDemo/functions/search/source.js

```diff
@@ -64,7 +65,9 @@ exports = function(query){
   }, {
     '$project': {
       'captain.name': 1, 
+      'captain.license': 1,
       'crew.name': 1, 
+      'crew.license': 1,
```

Comment:
adds license to crew and captain (in case the search result is a captain)

![image](https://user-images.githubusercontent.com/14609656/97764262-6c7e1500-1ae4-11eb-991a-2fddb77c1d10.png)